README.md

```diff
@@ -56,6 +57,7 @@ npm run start
 - `node src/index.js ./examples` - To read txt files from a directory and generate html from it to be stored in `./til` directory
 - `node src/index.js input.txt -o collection` / `node src/index.js input.txt -output collection` - To read a txt file and generate html from it to be stored in `./collection` directory
 - `node src/index.js ./examples -o collection` / `node src/index.js ./examples --output collection` - To read txt files from a directory and generate html from it to be stored in `./collection` directory
+- `node src/index.js --config path_to_your_config.toml` - To utilize your configuration file settings for conversion.
```

Comment:
Could you please add same example for `-c` in the same line?

Reply:
Solved in commit [09791](https://github.com/sdthaker/Node-TILify/pull/13/commits/09791bfc36a7c5b601649075c18c845a87c17d7c)